### DISCUSSION TITLE
0x02 Voxel 体素艺术与相关 NFT 的设计、开发与可能性探讨

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#77] [Workshop]
## 0x01 一句话简述

**主题：** Voxel 体素艺术与相关 NFT 的设计、开发与可能性探讨

**简介：** Minecraft 是 Voxel 体素艺术最成功的的案例，同时， Voxel 体素是一种和 Crypto 非常契合的艺术风格，因此在 NFT、元宇宙等场景中均有所发展。本期 Workshop 将会引导大家通过实践体会体素的魅力，同时介绍如何创体素转换为 dApp，并进行与之相关的发散性讨论。

一个 Voxel 的线上 Demo：

> https://welightproject.github.io/voxel-viewer-page

## 0x02 活动简介

- **活动时间： 2022 年 5 月 3 日晚 8:00 (HKT)**
- 活动地点：
  - 线上

- **DM（地下城主）：** 李大狗Leeduckgo

- **参与方式：**
  - NonceGeek 内部成员 —— 可直接参与
  - 持有 Web3DevNFT with Badges 的成员（部分做出过 Buidl 行动但还没有发放的同学也可以报名哦~） —— 可直接参与
  - 定向邀约成员/通道 —— 可直接参与
  - 除此之外的成员 —— 支付活动费后可参与

**参会费用：** 300 / 人

## 0x03 活动流程（简）

- Voxel x Generative Art based on Art —— 面向炫酷编程!  
- Code => Voxel => NFT + dApp 流程解析！ 
- Voxel 实践
- 自由脑暴
- Voxel to NFT 实践
- 二次脑暴 + 自由吹水——聊聊元宇宙、区块链相关的有的没的

**Comment by Weiyi1213** [comment]
Cool
